.ci/wercker.sh

```diff
@@ -57,6 +57,10 @@ no-error-xwiki)
   checkout_from https://github.com/xwiki/xwiki-commons.git
   cd .ci-temp/xwiki-commons
   git checkout 44b0c0048c516dae20cf5f8a71181af836549484
+  # The classes AnnotationUtility and CommonUtils have been renamed
+  # https://github.com/checkstyle/checkstyle/issues/5812
```

Comment:
this issue will be closed right after  merge.
please reference here link to PR to xwiki, to clearly define till what time hack should stay , comment should be like "till https:........"

Reply:
@pbludov , please look at my message in gitter.